### PR TITLE
Append created Students to Data service

### PR DESCRIPTION
**Pull request by dting**
- Create AbsenceRecord service to abstract http calls.
- Change AbsenceRecord api to use schoolId from body on POST.
- Send created Students in addition to created record in response to creating a new absence report.

@pdotsani

### CODE REVIEW

File: server/api/absence-record/absence-record.controller.js [hunk 1]
```diff
@@ -17,23 +17,6 @@ exports.index = function(req, res) {
...
-    AbsenceRecord.find({school: schoolID}, function(err, records) {
-      var result = records[0];
-      console.log(result);
-      return res.status(200).json(result);
-    });
-  });
-};
-
-/**
  * Get a single absence record
```
Comment (pdotsani) [inline server/api/absence-record/absence-record.controller.js]: Why was this deleted?

**Comment by pdotsani** [comment]
This LGTM!

File: server/api/absence-record/absence-record.controller.js (hunk 1 above)
Comment (dting) [inline server/api/absence-record/absence-record.controller.js]: It's not being used. Also looking up the school shouldn't be needed for getting the absence record. We can add this back when we need this route in the future.
